### PR TITLE
fix: truncate long floats in cli formatting for display

### PR DESCRIPTION
**Pull request by hemanth94**
Fixes the issue #1807 
Tried to truncate long floats to have 5 digits after decimal point.
Added Unit Test for a batch of floats. 
@universalmind303 I abandoned the previous logic of having a config parameter to control number of digits after decimal point, instead hardcoded it. Please suggest any corrections.

Thanks in advance.

### CODE REVIEW

File: crates/arrow_util/src/pretty.rs [hunk 1]
```diff
@@ -505,15 +505,36 @@ struct ColumnWidthSizeStats {
...
+                let floats_array = col
+                    .as_any()
+                    .downcast_ref::<Float64Array>()
+                    .unwrap()
+                    .values();
+
+                (0..floats_array.len())
+                    .map(|idx| {
+                        let s = fmt_floats(floats_array.get(idx));
+                        Ok(s)
```
Comment (tychoish) [inline crates/arrow_util/src/pretty.rs:520]: do you need to declare `s` or can you just `Ok(fmt_floats(<...>))`

File: crates/arrow_util/src/pretty.rs [hunk 2]
```diff
@@ -548,6 +569,17 @@ impl ColumnValues {
     }
 }
 
+fn fmt_floats(flt: Option<&f64>) -> String {
+    match flt {
+        Some(float) => {
+            //Change here to configure digits after decimalpoint for long floats
+            let str = format!("{:.5}", float);
+            str
```
Comment (tychoish) [inline crates/arrow_util/src/pretty.rs:577]: do you need to declare `str` here or can you just return the output of `format`?

**Review by tychoish — COMMENTED** [review]
why did you pick 5? should we indicate somehow that we've truncated the numbers in formatting?

File: crates/arrow_util/src/pretty.rs (hunk 1 above)
Comment (hemanth94) [reply crates/arrow_util/src/pretty.rs:520]: Yes I agree, My bad. I could have just written Ok(fmt_floats(<...>)). I will change this.

File: crates/arrow_util/src/pretty.rs (hunk 2 above)
Comment (hemanth94) [reply crates/arrow_util/src/pretty.rs:577]: Same issue with my code here, Will fix it.

Comment (hemanth94) [reply crates/arrow_util/src/pretty.rs:577]: I saw the sample floats mentioned in issue #1807  to pick 5 digits after decimals, I don't have a solid reason tbh.
I was hoping someone from the team would guide me as to what is required.

**Review by universalmind303 — APPROVED** [review]

**Pull request merged**